tests/components/metadata_item/test_views.py

```diff
@@ -367,3 +367,17 @@ async def test_list_metadata_items_returns_metadata_item_filtered_by_tags_and_ty
         received_total = body('.total').first()
         assert set(received_ids) == {str(create_file_1.id), str(create_file_2.id)}
         assert received_total == 2
+
+    async def test_list_metadata_items_returns_folder_with_dot_prefix(self, client, jq, metadata_item_factory):
+        created_metadata_item = await metadata_item_factory.create(type_=MetadataItemType.FOLDER, name='.test')
+
+        response = await client.get('/v1/metadata-items/')
```

Comment:
We should check here that filtering works:
```suggestion
        response = await client.get('/v1/metadata-items/', params={'name': '%.test'})
```